### PR TITLE
Update SentryOne Plan Explorer gallery version

### PR DESCRIPTION
**Pull request by s1cbriggs**
Releasing a newer preview of the SentryOne Plan Explorer extension (0.9.8) for Azure Data Studio.

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -2398,7 +2398,7 @@
 					},
 					"versions": [
 						{
-							"version": "0.9.7",
+							"version": "0.9.8",
 							"lastUpdated": "3/3/2020",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:2402]: Update the "lastUpdated" fields too

File: extensionsGallery-insider.json [hunk 2]
```diff
@@ -2413,7 +2413,7 @@
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
-									"source": "https://downloads.sentryone.com/downloads/ads/readme.0.9.7.md"
+									"source": "https://downloads.sentryone.com/downloads/ads/readme.0.9.8.md"
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:2416]: I get an error when trying this link - has the new readme been uploaded yet?

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

File: extensionsGallery-insider.json (hunk 2 above)
Comment (alanrenmsft) [inline extensionsGallery-insider.json:2416]: do you need to keep all versions of the readme file? if not you can simply have one file and make the version update easier.

**Review by alanrenmsft — APPROVED** [review]

File: extensionsGallery-insider.json (hunk 1 above)
Comment (s1cbriggs) [reply extensionsGallery-insider.json]: Good catch. Not sure how I missed that. Done. Thanks.

File: extensionsGallery-insider.json (hunk 2 above)
Comment (s1cbriggs) [reply extensionsGallery-insider.json:2416]: It has not yet. We typically publish the readme along with the extension. I can probably push up the readme if that would help.

Comment (s1cbriggs) [reply extensionsGallery-insider.json:2416]: No, probably not. I can address this next time we update the extension.

Comment (alanrenmsft) [reply extensionsGallery-insider.json:2416]: please publish it now and let me know, I will update the gallery files once I hear back from you.

**Review by Charles-Gagnon — APPROVED** [review]

File: extensionsGallery-insider.json (hunk 2 above)
Comment (s1cbriggs) [reply extensionsGallery-insider.json:2416]: readme.0.9.8.md has been published to our CDN.

https://downloads.sentryone.com/downloads/ads/readme.0.9.8.md

Our release date for the extension is Monday 10/19.

Comment (Charles-Gagnon) [reply extensionsGallery-insider.json:2416]: Ok, ping us when it's released and we can merge it in and publish the updated gallery at that time.

Comment (s1cbriggs) [reply extensionsGallery-insider.json:2416]: Version 0.9.8 of the PE ADS Extension has been published to our portal.

https://extensions.sentryone.com/

**Comment by Charles-Gagnon** [comment]
Merged and galleries updated - thanks!

**Pull request merged**